libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/components/dialogs/ConfirmationDialog.kt

```diff
@@ -39,18 +37,20 @@ import io.element.android.libraries.ui.strings.R as StringR
 fun ConfirmationDialog(
     title: String,
     content: String,
+    onSubmitClicked: () -> Unit,
+    onDismiss: () -> Unit,
     modifier: Modifier = Modifier,
     submitText: String = stringResource(id = StringR.string.ok),
     cancelText: String = stringResource(id = StringR.string.action_cancel),
     thirdButtonText: String? = null,
-    onSubmitClicked: () -> Unit = {},
-    onCancelClicked: () -> Unit = {},
+    emphasizeSubmitButton: Boolean = false,
```

Comment:
Added this to make the positive action 'bolder' than the rest, as per the Figma designs.